src/style.css

```
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+form {
+  position: relative;
+}
+
+.input {
+  font-family: Avenir, Arial, sans-serif;
```

```suggestion
  font-family: inherit;
```
Move font-family to the `html` tag selector